junit-vintage-engine/src/test/java/org/junit/vintage/engine/discovery/UniqueIdSelectorResolverTests.java

```diff
@@ -36,6 +37,7 @@
  */
 class UniqueIdSelectorResolverTests {
 
+	private Predicate<Class<?>> ALL_CLASSES = clazz -> true;
```

Comment:
Also, we don't actually use all caps for `static final` fields that are not _constants_ per se.

Rather, we prefer field names such as `allClassesPredicate` instead of `ALL_CLASSES`.